tests/copy_cut_paste.cpp

```diff
@@ -65,7 +65,7 @@ struct MockDataDeviceListener : DataDeviceListener
 };
 }
 
-TEST_F(CopyCutPaste, given_source_has_offered_data_sink_sees_offer)
```

Comment:
Alternatively, we could just move the bad buffer test to be the first one run and not have to disable any.